kubectl-fdb/cmd/plugin_version_checker.go

```diff
@@ -0,0 +1,126 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/hashicorp/go-retryablehttp"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+// KubectlFbReleaseURL is the public GirHub url we read the latest version
+const KubectlFbReleaseURL = "https://api.github.com/repos/FoundationDB/fdb-kubernetes-operator/releases/latest"
+
+// LocalTempVersionFileName is where we cache plugin version for 24 hours, so we don't call GitHub for every single command(also rate-limit issue on GitHub api calls)
+const LocalTempVersionFileName = "latest"
+
+// PluginVersionDetails Contains [partial] plugin version details from GitHub
+type PluginVersionDetails struct {
+	ID      int64  `json:"id"`
+	Version string `json:"tag_name"`
+	Name    string `json:"name"`
+}
+
+// It reads the latest fdb plugin version from local temp file,
+// if not exists, gets it from GitHub and store it locally and then returns it
+func getLatestPluginVersion() string {
+	fileName := filepath.Join(os.TempDir(), LocalTempVersionFileName)
```

Comment:
Can we document this behaviour? We should make sure that people understand that the plugin tries to write to the temp dir. We should also add an additional argument to the root command to overwrite the default directory used for storing this information (could be useful for unit testing the behaviour of these methods).

Reply:
Looks like we decided against the flag to define the cache directory?